### PR TITLE
fix(trading): rename mark price to price to make it spot compatible

### LINKED ISSUE #4214: List Spot Markets on the Markets Page
## Story

As a Console user
I want to be able to view Spot markets in the Markets page lists (i.e. differentiating them from Futures, Perpetuals etc. via a 'Type' column or similar)
So that I know they exist and can choose to view one 

## Acceptance Criteria

- [ ] I can see spot markets in the open markets tab
- [ ] I can see spot markets in the proposed markets tab
- [ ] I can see spot markets in the closed markets tab
- [ ] The markets are shown with a "SPOT" tag under the code instead of PERP or FUTR
- [ ] Change "Mark Price" to simply "Price" to be more generic and use the `lastTradedPrice` instead

## Tasks

- [ ] UX (if needed)
- [ ] Design (if needed)
- [ ] Team and stakeholder review
- [ ] Specs reviewed and created or adjusted
- [ ] Implementation
- [ ] Testing (unit and/or e2e)
- [ ] Code review
- [ ] QA review

## Sketch

Image

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes #4214 

# Description ℹ️

Rename mark price to price to make it spot compatible

# Demo 📺

Screenshot 2024-04-02 at 15 37 10

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

**Review by bwallacee — APPROVED** [review]
Approved. But will need a rebase for the e2e to run

File: apps/trading/client-pages/market/market-header-stats.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const MarketHeaderStats = ({ market }: MarketHeaderStatsProps) => {
 
   return (
     <>
-      <HeaderStat heading={t('Mark Price')} testId="market-price">
+      <HeaderStat heading={t('Price')} testId="market-price">
```
Comment (mattrussell36) [inline apps/trading/client-pages/market/market-header-stats.tsx:56]: I don't think we should change this one as my incoming Pr for handling spot markets in the header adds a new header stat for last price and it is label as such. So for futures and perps I think its preferable to have the label more accurately say 'Mark price'

File: apps/trading/client-pages/markets/closed.tsx [hunk 2]
```diff
@@ -216,7 +216,7 @@ const ClosedMarketsDataGrid = ({
         },
       },
       {
-        headerName: t('Mark price'),
```
Comment (mattrussell36) [inline apps/trading/client-pages/markets/closed.tsx:219]: We'll also need to actually render last trade for spots

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: apps/trading/client-pages/markets/closed.tsx (hunk 2 above)
Comment (MadalinaRaicu) [reply apps/trading/client-pages/markets/closed.tsx:219]: updated

**Review by guoguojin — APPROVED** [review]

**Pull request merged**